vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -0,0 +1,143 @@
+- `Bundle`: A Bundle is the deployment unit within an Entando Application. A Bundle can contain one or multiple components such as micro frontends, microservices, or any of the [component types](../../docs/ecr/ecr-bundle-details.md#overview) available in Entando. 
+- `Category`: Each Bundle Group belongs to a specific category. The initial possible categories are Solution Template, Packaged Business Capability (PBC), or Component Collection. An admin of an Entando Hub can refine the available categories as desired.
+- `Organization`: Bundle Groups belong to a single organization. Authors and managers can only update Bundle Groups for their own organization.  
+- `User`: User identity is managed within Keycloak, where users are granted roles within the Hub. Users must be assigned to a specific organization.
+
+Notes:
+- A private repository can be used for a Bundle, but this requires [an additional Kubernetes secret](../ecr/ecr-private-git-repo.md#overview) before deployment via the App Builder.
+
+### Roles
+
+Three roles are used to provide access to the Hub features:
+
+- `eh-author`: An author can create and edit Bundle Groups for their organization and submit them for publication.
+- `eh-manager`: A manager has the permissions of an author, but can also approve a publication request for their organization.
+- `eh-admin`: An admin has full access to create, update, and delete Bundle Groups for the entire Hub instance. An admin can also create categories and organizations, and assign users to an organization.
+- `guest`: Any user without one of the preceding roles is considered a guest in the Entando Hub and is given a read-only view of the public catalog. This is also true for unauthenticated users.
+
+### Bundle Group Version Status
+
+The possible statuses for the versions of a Bundle Group are:
+
+- `Draft`: This is the default status for the first version of a Bundle Group. 
+- `Publication Request`: An eh-author sets a version to this status to request an eh-manager or eh-admin to review the version and mark it for publication. An eh-manager or eh-admin may edit versions with this status.
+- `Published`: Versions with this status are visible in the home page list of available Bundle Groups and also available in the App Builder-facing API. An eh-manager or eh-admin may edit Published versions.
+- `Archived`: Previously Published versions are assigned this status. No edits can be made on an Archived version.
+- `Deletion Request`: An eh-manager or eh-admin can delete versions once this status has been set.
+
+Notes:
+- An eh-author can change any field except Organization while a version is in Draft.
+- There is no automated notification process when a Publication Request is made for a Bundle Group version.
+
+### Bundle Group Creation
+Clicking the "Add +" button at the top of the page generates the pop-up to create a new entry in the catalog:
```

Comment:
1. Back ticks like in the following one?
2. Maybe "... at the top of the page displays the form for creating a new Bundle Group"